README.md

```diff
@@ -1,4 +1,44 @@
 # bootstrap_go
 Lightweight bootstrap nodes for Golem network. Basic implementation in Go.
 
+# installation
+
+## prerequisites 
+- install `go` tools https://golang.org/doc/install  
+- create go workspace `mkdir ~/go`
```

Comment:
that's for unix based systems only. Might be okay to link to this page https://github.com/golang/go/wiki/SettingGOPATH

Reply:
ok